src/app.js

```diff
@@ -1 +1,25 @@
 'use strict';
+
+const fs = require('fs');
+
+const copyFile = (file, copy) => {
```

Comment:
Use more descriptive parameter names, such as 'source' and 'destination', instead of 'file' and 'copy'.